### ISSUE TITLE
Cannot install package

### ISSUE THREAD

**Issue by kirilloid** [lovell/sharp#724]
At first it required Python (there is nothing about it in package description BTW), but then it said it cannot install anyway.

```
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:27
6:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Windows_NT 10.0.14393
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-
gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\dev\node-img\node_modules\sharp
gyp ERR! node -v v6.9.2
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
```

**Comment by lovell** [comment]
Hello, the prerequisites are listed at http://sharp.dimens.io/en/stable/install/#prerequisites - Python is (currently) a dependency of node-gyp.

Are you able to share the results of running `npm install --verbose sharp`?

**Comment by kirilloid** [comment]
Initially I installed VS 2015 Express, but it didn't help much, I had been seeing the same error message. So I googled it found a thread with useful comment https://github.com/nodejs/node-gyp/issues/307#issuecomment-153450964 which helped me to move forward. Then I saw a pretty straightforward error message which advised installing Win 8.1 SDK and after that I successfully installed sharp package.

So I'm closing the issue, but I suggest you explaining how to install it on windows in just a little more detailed fashion, even though it's gyp requirements and they're the same for various projects:
- install `python` (add to path)
- install [vs express](https://www.visualstudio.com/vs/visual-studio-express/) e.g. 2015 (make sure Win 8.1 SDK option is ticked).
- run `npm config set msvs_version 2015 --global` to set it to make gyp using proper version.
